### PR TITLE
feat: add fetch config from internet

### PR DESCRIPTION
**Pull request by gelleson**
Add feature to fetch config from internet

### CODE REVIEW

File: cmd/easeprobe/main.go [hunk 1]
```diff
@@ -31,17 +32,24 @@ import (
...
+		return value
+	}
+	return defaultValue
+}
+
+func isExternalLink(value string) bool {
+	_, err := url.Parse(value)
+	return err == nil
+}
+
```
Comment (haoel) [inline cmd/easeprobe/main.go:46]: do you still need this function in `main.go`? as I've seen a similar function is `conf.,go`

File: cmd/easeprobe/main.go [hunk 2]
```diff
@@ -31,17 +32,24 @@ import (
...
 func main() {
 
-	dryNotify := flag.Bool("d", false, "dry notification mode")
-	yamlFile := flag.String("f", "config.yaml", "configuration file")
+	dryNotify := flag.Bool("d", os.Getenv("DRY") == "true", "dry notification mode")
+	yamlFile := flag.String("f", getEnvOrDefault("CONFIG", "config.yaml"), "configuration file")
```
Comment (haoel) [inline cmd/easeprobe/main.go:50]: How about using the `PROBE_` prefix for the environment variable? this could avoid variable name conflicts.

**Comment by haoel** [comment]
@gelleson  do you have any update on this?

**Comment by gelleson** [comment]
@haoel I have updated my PR.

**Review by haoel — APPROVED** [review]
LGTM

**Pull request merged**